### ISSUE TITLE
 'NearestNeighbors' is not defined

### ISSUE THREAD

**Issue by eduamf** [VarIr/scikit-hubness#65]
Running ...
```
# assess the actual degree of hubness in dexter
from skhubness import Hubness
hub = Hubness(k=10, metric='cosine')
hub.fit(X)
```

X.shape = (300, 20000), y.shape=(300,)
NameError: name 'NearestNeighbors' is not defined

**Comment by eduamf** [comment]
The previous error was about the scikit-learn versions bigger than 0.22.
 
After fix the scikit-learn version to 0.21.3, I closed Spyder and had to rebuild from source.

**Comment by eduamf** [comment]
The previous error was about the scikit-learn versions bigger than 0.22.

After fix the scikit-learn version to 0.21.3, I closed Spyder and had to rebuild from source.
